### PR TITLE
v1.6.2

### PR DESCRIPTION
**Pull request by zomars**
- #2245
- #2379
- #2485
- #2486
- #2506
- #2553
- #2602
- #2652
- #2654
- #2658
- #2665
- #2669
- #2671
- #2674
- #2696
- #2707
- #2710
- #2718
- #2719
- #2722
- #2723
- #2725
- #2727
- #2728
- #2729
- #2730
- #2732
- #2737
- #2743
- #2745
- #2746
- #2747
- #2752
- #2755
- #2756
- #2761
- #2764
- #2768
- #2769
- #2770
- #2773
- #2777
- #2779
- #2782
- #2783
- #2784
- #2788
- #2790
- #2791
- #2796
- #2801
- #2803
- #2804
- #2805
- #2807
- #2808
- #2810
- #2812
- #2819
- #2820
- #2821
- #2828
- #2832
- #2836
- #2840
- #2842
- #2843
- #2848
- #2849
- #2860
- #2862
- #2866
- #2870
- #2872
- #2879
- #2880
- #2882
- #2883
- #2885
- #2886
- #2887
- #2889
- #2891
- #2892
- #2893
- #2894
- #2895
- #2904
- #2905
- #2908
- #2909
- #2912
- #2919
- #2921
- #2924
- #2925

### CODE REVIEW

File: apps/web/ee/components/support/HelpMenuItem.tsx [hunk 1]
```diff
@@ -1,13 +1,191 @@
...
+
+import classNames from "@lib/classNames";
+import { trpc } from "@lib/trpc";
+
+import ContactMenuItem from "./ContactMenuItem";
 
 export default function HelpMenuItem() {
+  const [rating, setRating] = useState<null | string>(null);
+  const [comment, setComment] = useState("");
+  // const [errorMessage, setErrorMessage] = useState(false);
```
Comment (alannnc) [inline apps/web/ee/components/support/HelpMenuItem.tsx:16]: We should remove this line

File: apps/web/pages/api/book/event.ts [hunk 2]
```diff
@@ -637,11 +681,18 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
...
+      evt,
+      originalRescheduledBooking.uid,
+      booking?.id,
+      reqBody.rescheduleReason
+    );
     // This gets overridden when updating the event - to check if notes have been hidden or not. We just reset this back
     // to the default description when we are sending the emails.
     evt.description = eventType.description;
 
+    console.log("🚀 ~ file: event.ts ~ line 684 ~ handler ~ descripiton", reqBody);
```
Comment (alannnc) [inline apps/web/pages/api/book/event.ts:694]: I think this is no longer needed

**Review by alannnc — APPROVED** [review]
Working on my end and finished code review.

**Review by leog — APPROVED** [review]
LGTM 🚀

**Pull request merged**